crates/ruff_wasm/tests/api.rs

```diff
@@ -23,7 +23,7 @@ macro_rules! check {
 #[wasm_bindgen_test]
 fn empty_config() {
     check!(
-        "if (1, 2):\n pass",
```

Comment:
Are the changes in this file intentional?

Reply:
Yeah, because these rules are now enabled in debug here (whereas before, I don't think we were running with `--all-features`). So without these changes, we get additional pycodestyle errors.